test/fixtures/content/page-metadata-jsonld-error.html

```diff
@@ -0,0 +1,16 @@
+<head>
+    <title>Home | Helix Project Boilerplate</title>
+    <link rel="canonical" href="https://helix-pages.com/page-metadata-jsonld-error">
+    <meta property="og:title" content="Home | Helix Project Boilerplate">
+    <meta property="og:url" content="https://helix-pages.com/page-metadata-jsonld-error">
+    <meta property="og:image" content="https://helix-pages.com/default-meta-image.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <meta property="og:image:secure_url" content="https://helix-pages.com/default-meta-image.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:title" content="Home | Helix Project Boilerplate">
+    <meta name="twitter:image" content="https://helix-pages.com/default-meta-image.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
+    <script type="application/javascript">// error in json-ld: Unexpected non-whitespace character after JSON at position 10</script>
```

Comment:
is this useful? or maybe better as html comment:
```
<!-- error in json-ld: Unexpected non-whitespace character after JSON at position 10 -->
```

Reply:
i think it is useful to know that there was an error, but i would leave it at that without any details.

Reply:
with the erronous JSON ?

Reply:
yeah that sounds great... but i would keep the `<script>` tag empty and the error short

```
<script 
  type="application/ld+json" 
  data-error="error in json-ld">
</script>
```